_data/accessibility-tests/banner.yml

```diff
@@ -0,0 +1,112 @@
+title: Banner
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey meaning in the banner.
+    summary_additional: When viewing the banner, you never have to rely on color alone to understand meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Banner appears consistently.
+    summary_additional: The banner appears in the same location on every page of your government website.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.4
+  - summary: Icons meet color contrast requirements.
+    summary_additional: |
+       When you open the banner and use ANDI or another color contrast analyzer to look at the hex numbers,
+       contrast between the icons and the background is at least 4:5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.11
+# Zoom/screen magnification tests
+  - summary: Text retains functionality when screen is magnified.
+    summary_additional: |
+      When you zoom in to 200%, text in the banner remains visible and retains all of its functionality.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.4
+  - summary: Banner content is visible and functional when screen is magnified.
+    summary_additional: |
+      When you zoom in to 400%,
+      banner content adjusts to fit the width of the screen without losing functionality.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Banner can be accessed, opened, and closed with only a keyboard.
+    summary_additional: |
+      When you navigate your site with a keyboard,
+      you can access the banner by using `tab` to move forward or `tab` + `shift` to move backwards.
+      You can use the `enter` and `spacebar` keys to open and close the banner.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+  - summary: Banner does not trap keyboard focus.
+    summary_additional: |
+      When you navigate with a keyboard, you can easily move into and out of the banner,
+      regardless of whether the banner is open or closed.
+      When you navigate out of an open banner using your keyboard, it doesn't automatically close.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.2
+  - summary: Banner is the first interactive object after the main navigation.
```

Comment:
Curious - is this an accessibility test? It feels more like it would fall under something unrelated to a11y like guidance/standards compliance instead.